coding/cpp_style_guide.rst

```diff
@@ -84,10 +84,17 @@ These tasks are made much simpler if the code is easily readable and well-docume
 
 .. _style-guide-cpp-2-2:
 
-2-2. We are writing C++11/14 with some restrictions.
-----------------------------------------------------
+2-2. We are writing C++14
+-------------------------
 
-The official policy on the use of C++11 features is at :ref:`Policy on use of C++11/14 language features <style-guide-cpp-cpp-11-14>`.
+The C++11 standard and the C++14 improvements to it bring a number of useful language features that make the resulting code more expressive, easier to read, and safer.
+
+We follow the official: International Standard ISO/IEC 14882:2014(E) – Programming Language C++, without any compiler specific extensions.
+
+.. seealso::
+
+   - :ref:`pipelines:source-install-redhat-legacy` from the `LSST Science Pipelines <https://pipelines.lsst.io>`__ documentation.
```

Comment:
While this used to be sufficient, the docs for 14.0 that specify a minimum of gcc 4.8 are _not_ sufficient now.  We should make that clear.